ParcelDistributionCenter/ParcelDistributionCenter.Model/Entites/Courier.cs

```diff
@@ -35,9 +35,11 @@ public Courier()
         public string CourierJsonId { get; init; }
 
         // TODO: Think if nullable values are needed
+        [NotMapped]
```

Comment:
Czemu tutaj NotMapped? Wydaje mi się, że powinno być mapwoane przez EntityFramweork

Reply:
Tutaj generalnie nie rozumiem czemu mammy DeliveryMachine w modelu Courier?
Chyba w poprzednim PR to doszło ale nie widzę zastosowania żadnego połączenia Kuriera z Paczkomatem?